components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/EnvironmentConfigurations.java

```diff
@@ -0,0 +1,69 @@
+package org.wso2.carbon.apimgt.core.configuration.models;
+/*
```

Comment:
Add the license header on top of the page. Put  "package org.wso2.carbon.apimgt.core.configuration.models;" after the license header